packages/node/src/lib/test-verify-webhook.ts

```diff
@@ -0,0 +1,90 @@
+import crypto from 'crypto';
+
+import fetch, { Headers } from 'node-fetch';
+
+async function verifyWebhook(url: string, email: string, secret: string, opts = { unsigned: false }) {
+  if (!url || !email || !secret) {
+    throw new Error('Missing required params');
+  }
+
+  const time = Date.now();
+  const payload = {
+    email,
+  };
+
+  const headers = new Headers({
+    'User-Agent': 'readme',
```

Comment:
might want to give this some more identifable data like the SDK version or `proces.arch` and `process.platform`. Believe there's work in the main SDK to generate this that you should be able to reuse.